test/unit/definitions/divorce/UserEventAuthorisation.test.js

```diff
@@ -1,7 +1,7 @@
 const expect = require('chai').expect;
 const { differenceWith, intersectionWith, concat } = require('lodash');
 
-const AuthorisationCaseEvent = Object.assign(require('definitions/divorce/json/AuthorisationCaseEvent'), []);
+const AuthorisationCaseEvent = Object.assign(require('definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent'), []);
```

Comment:
Why do we have a repeated subsection named the same thing as its containing dir?